.github/workflows/pr_report.yml

```diff
@@ -0,0 +1,44 @@
+name: Generate Report on Eyecite PR
+on:
+  pull_request:
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: snok/install-poetry@v1
+        with:
+          version: 1.1.13
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+
+      - name: Install Poetry and run Bash script
```

Comment:
This name is wrong.

Reply:
the install poetry?  part.  Install dependencies in poetry and run benchmarks.?